modules/application/src/main/java/edu/northeastern/truthtree/adapter/utilities/URLUtil.java

```diff
@@ -54,4 +62,49 @@ private static String readURL(String url) {
 
     return response.toString();
   }
+
+  /**
+   * Reads a web page and turns its contents into a string.
+   *
+   * @param url The web address of the web page to be read and converted.
+   * @return Web contents as a string
+   */
+  public static String postJSONFromURL(String url, String jsonString) {
```

Comment:
Easier to support post methods for any DTO and that's why I added this method in URLUtil class. I am just converting any DTO into its String equivalent and sending stringJson to the util method. In case of using rest template, I will have to put all the request fields of json in a hashmap which tedious and specific to each dtos and hence this generic method cannot be used.

Reply:
Even then, `RestTemplate` got you covered, the following method does the exact same thing with very minimal effort.

```
public static String postJSONFromURL(String url, String jsonString) {
  HttpHeaders headers = new HttpHeaders();
  headers.setContentType(MediaType.APPLICATION_JSON);

  HttpEntity<String> entity = new HttpEntity<String>(jsonString, headers);
  ResponseEntity<String> response = restTemplate. postForEntity(url, entity, String.class);
  return response.getBody()
}
```

Reply:
I'll add this change in next commit. Same thing will have to be updated for get method added in last sprints.